src/status_im/ui/screens/wallet/manage_connections/views.cljs

```diff
@@ -41,12 +57,18 @@
                                     :height 20
                                     :container-style {:elevation 3}
                                     :color colors/red}]]
-
         (when selected-account ;; The account might not be available in theory, if deleted
-          [rn/view {:style (styles/selected-account-container (:color selected-account))}
+          [rn/touchable-opacity {:style (styles/selected-account-container (:color selected-account))
+                                 :on-press #(swap! show-account-selector? not)}
            [rn/text {:style styles/selected-account} (:name selected-account)]])]]]]))
 
 (defn views []
-  (let [legacy-sessions @(re-frame/subscribe [:wallet-connect-legacy/sessions])]
+  (let [legacy-sessions @(re-frame/subscribe [:wallet-connect-legacy/sessions])
+        visible-accounts @(re-frame/subscribe [:visible-accounts-without-watch-only])]
     [rn/view {:margin-top 10}
-     (doall (map print-session-info legacy-sessions))]))
+     (doall (map-indexed (fn [idx session]
```

Comment:
better to use flat-list ?

Reply:
but im not sure, just a question

Reply:
Will try a flat list

Reply:
I don't think using a flat list is achievable here .. Because I need to show the bottom sheet per row item outside the list component

Reply:
Used a flat list